### PR TITLE
[TASK] Switch to PHP-based Documentation Rendering

### PR DESCRIPTION
**Pull request by Alagts**
Use 
```shell
make docs
```
to render the Documentation files locally.
I deleted the menu entries "Configuration/Index", "Breaking/Index" and "Testing/Index" since they did not appear in the document tree. 

Resolves: #39

Releases: 4.0.0

### CODE REVIEW

File: Documentation/Index.rst [hunk 1]
```diff
@@ -51,6 +51,3 @@ a GNU/GPL CMS/Framework available from http://typo3.org
 
    Introduction/Index
    Installation/Index
-   Configuration/Index
```
Comment (linawolf) [inline Documentation/Index.rst:54]: Why do you remove these pages from the menu?

Comment (Alagts) [reply Documentation/Index.rst:54]: When I used make docs then a warning appeared: [2024-10-15T14:08:28.055255+00:00] app.WARNING: Menu entry "Configuration/Index" was not found in the document tree. Ignoring it.
So I concluded that these entries can be deleted safely.

Comment (Alagts) [reply Documentation/Index.rst:54]: The writer has not created an Index.rst file in folder Configuration, which was also not there. So I deleted this from the menu.

Comment (linawolf) [reply Documentation/Index.rst:54]: Ok, seems like they never existed

**Review by linawolf — APPROVED** [review]
Looks good to me

**Pull request merged**